src/js/firstparties/twitter.js

```diff
@@ -80,6 +93,26 @@ function unwrapSpecialTwitterURLs() {
   });
 }
 
-unwrapSpecialTwitterURLs();
+function unwrapSpecialTwitterPwaURLs() {
+  try {
+    // iterate users object (usually should only be one, with some random ID?)
+    for (const entityKey of Object.keys(window.__INITIAL_STATE__.entities.users.entities)) {
+      const entityValue = window.wrappedJSObject.__INITIAL_STATE__.entities.users.entities[entityKey];
```

Comment:
I don't think we should make use of `wrappedJSObject` at all as it's a non-standard property that's not supported in any modern browser at this point.

Reply:
If we want to escape the separate execution context that content scripts operate in, we should instead have the content script inject a script into the page ([for example](https://github.com/EFForg/privacybadger/blob/8831ac842100487e15482baa71ade464b3dd7535/src/js/contentscripts/dnt.js#L56)).

Reply:
Sooo, looked again at why this is done, and the reason is the XRayVision in Firefox, see https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Xray_vision

And no, that script is already injected. I could not find any documentation for `injectScript` . I guess it's just an old alias for [`executeScript`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/executeScript) or so. It also seems to be some bad eval or so and is not listed in this functions here, you can usually use to clone objects or functions into the site etc: https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Language_Bindings/Components.utils

Reply:
[`injectScript()` is defined in another content script](https://github.com/EFForg/privacybadger/blob/18feb68d8a5d50a6209dbd24851283f922db71c2/src/js/contentscripts/utils.js#L24).